### PR TITLE
✨(marsha) cronjob running clean_medialive_dev_stack command

### PR DESCRIPTION
**Pull request by lunika**
## Purpose

run command clean_medialive_dev_stack added in PR #1108
https://github.com/openfun/marsha/pull/1108

## Proposal

- [x] add cronjob running clean_medialive_dev_stack command

### CODE REVIEW

File: apps/marsha/templates/services/app/cronjob_clean_medialive_dev_stack.yml.j2 [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+                - name: DJANGO_CONFIGURATION
+                  value: "{{ marsha_django_configuration }}"
+                - name: POSTGRES_DB
+                  value: "{{ marsha_postgresql_database }}"
+                - name: POSTGRES_HOST
+                  value: "marsha-{{ marsha_postgresql_host }}-{{ deployment_stamp }}"
+                - name: POSTGRES_PORT
+                  value: "{{ marsha_postgresql_port }}"
+                - name: DJANGO_ALLOWED_HOSTS
+                  value: "{{ marsha_host | blue_green_hosts }}"
```
Comment (sampaccoud) [inline apps/marsha/templates/services/app/cronjob_clean_medialive_dev_stack.yml.j2]: Don't set it as this is a cron job?

File: apps/marsha/templates/services/app/cronjob_clean_medialive_dev_stack.yml.j2 [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+                - name: DJANGO_CLOUDFRONT_PRIVATE_KEY_PATH
```
Comment (sampaccoud) [inline apps/marsha/templates/services/app/cronjob_clean_medialive_dev_stack.yml.j2]: Don't set it as this is a cron job?

**Review by sampaccoud — APPROVED** [review]

File: apps/marsha/templates/services/app/cronjob_clean_medialive_dev_stack.yml.j2 (hunk 2 above)
Comment (lunika) [reply apps/marsha/templates/services/app/cronjob_clean_medialive_dev_stack.yml.j2]: :+1:

File: apps/marsha/templates/services/app/cronjob_clean_medialive_dev_stack.yml.j2 (hunk 1 above)
Comment (lunika) [reply apps/marsha/templates/services/app/cronjob_clean_medialive_dev_stack.yml.j2]: :+1:

**Pull request merged**